system-test/tpc-tests.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {after, describe, it} from 'mocha';
+import {Bigtable} from '../src';
+
+describe.skip('TPC tests', () => {
```

Comment:
re: chat comment, I'm not sure if TPC is an accepted term to use for this stuff; might want to double check with @bshaffer.

Reply:
It is not, please remove it!

Reply:
This was done earlier